integreat_cms/cms/utils/welcome_mail_utils.py

```diff
@@ -77,5 +81,35 @@ def send_welcome_mail(request, user, activation):
         logger.exception(e)
         messages.error(
             request,
-            _("An error occurred! Could not send {}.").format(debug_mail_type),
+            __(
+                error_occured,
+                _("Malformed header data."),
+            ),
+        )
+    except SMTPAuthenticationError as e:
+        logger.exception(e)
```

Comment:
```suggestion
        logger.error(e)
```
I think with these specific error messages, we don't need the whole stack trace from the smtp library, just the error is enough.